src/js/view/controls/shortcuts-tooltip.js

```diff
@@ -90,28 +92,34 @@ export default function (container, api, model) {
             api.play(settingsInteraction);
         }
     };
+
+    const destroy = () => {
+        close();
+    };
```

Comment:
Do the model callbacks also need to be removed here? I noticed they are removed in info-overlay, but they are already removed in controls.js when disable is called.

Reply:
if there is an execution path where we could be hitting those callback after a destroy then i would say yes. if the info-overlay removal is a superset of the destroy here and will always be the entrypoint for destruction of this component i'd say we're okay not removing them here.

thoughts @robwalch ?

Reply:
I don't any listeners added to `model` in this module. Just calls to `get` and `set`...

In general, yes. If a module adds listeners to the anything, it should remove them when destroyed. So if you use `model.change/on/once` pass a context like `this` and then remove all listeners for that on destroy  `model.off(null, null, this)`